ocw/lib/EC2.py

```diff
@@ -334,3 +355,42 @@ def cleanup_images(self, valid_period_days):
                     else:
                         self.log_dbg("Delete image '{}' (ami:{})".format(img['Name'], img['ImageId']))
                         self.ec2_client(region).deregister_image(ImageId=img['ImageId'], DryRun=False)
+
+    def cleanup_k8s_jobs(self):
+        self.create_credentials_file()
+        self.cmd_exec("aws sts get-caller-identity")
+
+        clusters = {}
+        for region in self.cluster_regions:
+            response = self.eks_client(region).list_clusters()
+            if 'clusters' in response and len(response['clusters']) > 0:
+                clusters[region] = []
+                for cluster_name in response['clusters']:
+                    client = self.kubectl_client(region, cluster_name)
+                    if client is not None:
```

Comment:
let's add `else:` to this "if" . and log cases when we failed to get client

Reply:
But the log is already set in the kubectl_client method:
`self.log_err(f"Cannot get the kubeconfig for the cluster {cluster_name} on region {region}")`

Reply:
ah you are right